packages/demo/components/DetailComponent/ShowEntity.tsx

```diff
@@ -18,13 +18,20 @@ import {
   Pressable,
 } from "react-native";
 
+import { Feather } from '@expo/vector-icons';
```

Comment:
Hi @rajatkhade 

Donot import like this use normal way import all package all at one line

Reply:
Hi @nihal772  @rajatkhade 

Check with this code I guess you need to move this code to TSD folder